decidim-core/lib/decidim/core/api/user_type.rb

```diff
@@ -19,5 +19,9 @@ module Decidim
     field :organizationName, !types.String, "The user's organization name" do
       resolve ->(obj, _args, _ctx) { obj.organization.name }
     end
+
+    field :isAuthorized, !types.Boolean, "Whether the user is authorized or not." do
+      resolve ->(obj, _args, _ctx) { obj.authorized? }
```

Comment:
Wh are we using authorized here? Also, why are we adding `isAuthorized` if it's not used anywhere?